raiden/network/blockchain_service.py

```diff
@@ -212,7 +213,7 @@ def service_registry(self, address: Address) -> ServiceRegistry:
 
         return self.address_to_service_registry[address]
 
-    def payment_channel(self, canonical_identifier: CanonicalIdentifier) -> PaymentChannel:
+    def payment_channel(self, participant1: Address, canonical_identifier: CanonicalIdentifier) -> PaymentChannel:
```

Comment:
i don't think it makes much sense to call this `participant1` if it's the only participant parameter in this signature.

do we have any alternatives to this name? `creator_address`?